app/views/courses/CoursesView.coffee

```diff
@@ -112,6 +112,18 @@ module.exports = class CoursesView extends RootView
       @ownerNameMap[ownerID] = NameLoader.getName(ownerID) for ownerID in ownerIDs
       @render?()
     )
+    @allCompleted = true
+    for classroom, index in @classrooms.models
```

Comment:
Is there any syntax sugar that we can implement it simpler

Reply:
Unsure if this can be done in a nicer way, because you are going "depth first" here. You are trying to find the any `stats.courseComplete` that is false, and stop at the first one. 

You could potentially use array methods like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find to make it a little cleaner, but it would still be a depth first traversal of these arrays.

Reply:
@wjllance @smallst I haven't tested this code, but I would try something like this:
```coffeescript
    @allCompleted = not _.some @classrooms.models, classroom ->
      _.some @courseInstances.where({classroomID: classroom.id}), courseInstance ->
        course = @store.state.courses.byId[courseInstance.get('courseID')]
        stats = classroom.statsForSessions(courseInstance.sessions, course._id)
        not stats.courseComplete
```
The Lodash `_.some` method will implement this behavior of stopping as soon as it finds a match, so it's a bit higher level than implementing this with `for ... break`.